pkg/test/framework/features/features.yaml

```diff
@@ -61,15 +61,51 @@ features:
     user:
     ingress:
       mtls:
+        server-keyrotation:
```

Comment:
```server-keyrotation:
        generic-compoundrotation:
        nongeneric-compoundrotation:
        gateway:
          invalid-secret:
          valid-secret:
```
These are not features, they are test cases. Please leave them in the tests, don't add into this file.